docs/contribute.md

```diff
@@ -1,10 +1,45 @@
+# Table of Contents
+- [Contributing](#contributing)
+- [PR Submission](#pr-submission)
+- [Prerequisites: Node and Npm](#prerequisites-node-and-npm)
+- [Install Dependencies](#install-dependencies)
+- [Build and Test](#build-and-test)
+  * [Build All Tasks (this can take a while):](#build-all-tasks-this-can-take-a-while)
+  * [Build a specific task (recommended):](#build-a-specific-task-recommended)
+  * [Run Tests](#run-tests)
+  * [Legacy Tests](#legacy-tests)
+- [Package](#package)
+
 # Contributing
 
 This repo contains the in-the-box tasks for Team Services build. Tasks in this repo get deployed every three weeks to Team Services and appear in TFS quarterly updates.
 
 If you are creating tasks that you believe the community can benefit from consider [creating an extension](https://www.visualstudio.com/integrate/extensions/develop/add-build-task).
 
-## Node and Npm
+# PR Submission
+
+We welcome contributions to the project!  To help us get your PR through the review process and give it the attention it deserves please follow these steps during submission:
+- There are many different teams working in this repository each of whom may be responsible for one or more tasks.  You can help them respond to your PR by assigning a the label to your PR that matches the `category` in your changed task's `task.json`.  Notice the `category` in the snippet from a `task.json` below:
+```json
+{
+    "id": "333b11bd-d341-40d9-afcf-b32d5ce6f23b",
+    "name": "NuGetCommand",
+    "friendlyName": "NuGet",
+    "description": "Restore, pack, or push NuGet packages, or run a NuGet command. Supports NuGet.org and authenticated feeds like Package Management and MyGet. Uses NuGet.exe and works with .NET Framework apps. For .NET Core and .NET Standard apps, use the .NET Core task.",
+    "helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?LinkID=613747)",
+    "category": "Package",
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 2,
+        "Minor": 0,
+        "Patch": 21
+    },
+```
+- Assign a reviewer.  Look in GIT history for your file and find either the creator or the most prolific contributor and assign them as a reviewer.  If they can't review your PR they'll should be able to find someone who can.
```

Comment:
typo `they'll` should be `they` in this context

Reply:
@ericsciple, fixed.